app/Http/Requests/StoreSettingsProfile.php

```diff
@@ -25,6 +25,8 @@ public function rules()
     {
         $rules = [
             'name' => 'required|string|min:2',
+            'pronouns' => 'string|max:45',
+            'link' => 'url|max:90|regex:/^https?:\/\/(www\.)?beacons\.ai\/[a-zA-Z0-9\-_.]+$/',
```

Comment:
DOn't limit to beacons.ai, they aren't paying for this feature. It can be any valid url